web/src/search/input/MonacoQueryInput.tsx

```
@@ -214,9 +243,13 @@ export class MonacoQueryInput extends React.PureComponent<MonacoQueryInputProps>
             )
         )
 
-        // Disable some default Monaco keybindings
-        for (const keybinding of NOOP_KEYBINDINGS) {
-            editor.addCommand(keybinding, noop)
+        // Disable default Monaco keybindings
+        if (hasKeybindingService(editor)) {
```

How do we want to handle the scenario of Monaco changing something that would break this, given they are private functions? E.g. renaming one of the properties.

Ideally I think we would have it so that tests fail on the PR that upgrades Monaco, but currently this would silently pass (with the shortcuts broken again). Maybe we should throw an Error if `hasKeybindingService()` returns `false`?

---

Done in 9e04265